docs-site/content/ja/db/getting-started.md

```
@@ -0,0 +1,138 @@
+1. **データベースエンジン** — 使用中のデータベース (PostgreSQL、MySQL、SQLite、MongoDB など)
+2. **ORM/クエリビルダー** — データアクセスレイヤーツール
+3. **マルチテナンシー戦略** — シングルスキーマ、テナント単位スキーマ、テナント単位DB、またはなし
+4. **マイグレーションツール** — スキーマ変更管理ツール
+
+各質問に対して適切なオプションを選択します。
+
+### ステップ 4: 生成されたファイルを確認
+
+初期化後、`.moai/project/db/` ディレクトリに以下のファイルが作成されます:
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)